File: packages/firestore/ios/Plugin/Classes/Options/GetDocumentOptions.swift
import Foundation

@objc public class GetDocumentOptions: NSObject {
    private var reference: String
    private var databaseId: String?

    init(reference: String, databaseId: String? = nil) {
        self.reference = reference
        self.databaseId = databaseId
    }

    func getReference() -> String {
        return reference
    }

    func getDatabaseId() -> String? {
        return databaseId
    }
}

File: packages/firestore/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/firestore/classes/options/GetDocumentOptions.java
package io.capawesome.capacitorjs.plugins.firebase.firestore.classes.options;

import androidx.annotation.Nullable;

public class GetDocumentOptions {

    private String reference;
    @Nullable
    private String databaseId;

    public GetDocumentOptions(String reference, @Nullable String databaseId) {
        this.reference = reference;
        this.databaseId = databaseId;
    }

    public String getReference() {
        return reference;
    }

    @Nullable
    public String getDatabaseId() {
        return databaseId;
    }
}
